### DISCUSSION TITLE
SPEARHEAD v1.0.0

### DISCUSSION THREAD

**Discussion by Nyemse** [Nyemse/SPEARHEAD#54] [General]
First public release!

<hr /><em>This discussion was created from the release <a href='https://github.com/Nyemse/SPEARHEAD/releases/tag/'>SPEARHEAD v1.0.0</a>.</em>

**Comment by MEEP272727** [comment]
Great looking system, however i can't get it to work even with all animations reuploaded and put in, the problem is I have two gery arms sticking out in front of my on my avatar, and when I select a gun tool nothing happens, any help?
